controllers/user.js

```diff
@@ -6,7 +6,11 @@ module.exports.get = async function get (req, res) {
 }
 
 module.exports.getOne = async function getOne (req, res) {
-  const data = await UserModel.findOne({ _id: req.params.id })
+  const data = await UserModel.findById(req.params.id)
+  if (data === null) {
+    res.status(200).send('This id is not stored in the server')
+    return
+  }
```

Comment:
you can simplify this if statement to just one line:

`if (!data) return res.status(200).send('This id is not stored in the server')`

Reply:
done, thanks!